ubcpi/static/html/ubcpi.html

```diff
@@ -1,6 +1,6 @@
 <div class="ubcpi_block problem">
 
-    <div class="revise" data-ng-controller="ReviseController as rc">
+    <span class="revise" data-ng-controller="ReviseController as rc">
```

Comment:
It looks like (just) the opening tag was automatically changed by my IDE when I was implementing portions of the progress bar that contained <span> tags. I have reverted this to a <div>.